router/src/lib.rs

```diff
@@ -205,10 +214,71 @@ pub(crate) struct ChatCompletion {
 pub(crate) struct ChatCompletionComplete {
     pub index: u32,
     pub message: Message,
-    pub logprobs: Option<Vec<f32>>,
```

Comment:
@drbh why was this vec f32?

Reply:
whoops silly mistake, thanks for resolving in this PR 🙏